package.json

```diff
@@ -18,7 +18,8 @@
   "dependencies": {
     "@alephium/token-list": "^0.0.12",
     "@alephium/walletconnect-provider": "^0.27.2",
-    "@alephium/web3": "^0.27.2"
+    "@alephium/web3": "^0.27.2",
+    "@yornaath/batshit": "^0.9.0"
```

Comment:
Is this being used in more than 1 app?

Reply:
Yes... BUT I'm unable to modify this or the type check starts failing again...

Reply:
Fixed in a follow up PR!